websites/S/Shard Hosting/presence.ts

```diff
@@ -0,0 +1,130 @@
+            presenceData.details = `Managing ${serverName[0]}`;
+            presenceData.state = "Databases";
+        } else if (document.location.pathname.includes("schedules")) {
+            const serverName = document.title.split(" | ");
+            presenceData.details = `Managing ${serverName[0]}`;
+            presenceData.state = "Schedules";
+        } else if (document.location.pathname.includes("users")) {
+            const serverName = document.title.split(" | ");
+            presenceData.details = `Managing ${serverName[0]}`;
+            presenceData.state = "Subusers";
+        } else if (document.location.pathname.includes("backups")) {
+            const serverName = document.title.split(" | ");
+            presenceData.details = `Managing ${serverName[0]}`;
+            presenceData.state = "Backups";
+        } else if (document.location.pathname.includes("subdomains")) {
+            const serverName = document.title.split(" | ");
+            presenceData.details = `Managing ${serverName[0]}`;
+            presenceData.state = "Subdomains";
+        } else if (document.location.pathname.includes("network")) {
+            const serverName = document.title.split(" | ");
+            presenceData.details = `Managing ${serverName[0]}`;
+            presenceData.state = "Network";
+        } else if (document.location.pathname.includes("startup")) {
+            const serverName = document.title.split(" | ");
+            presenceData.details = `Managing ${serverName[0]}`;
+            presenceData.state = "Startup";
+        } else if (document.location.pathname.includes("settings")) {
+            const serverName = document.title.split(" | ");
+            presenceData.details = `Managing ${serverName[0]}`;
+            presenceData.state = "Settings";
+        } else if (document.location.pathname.includes("server")) {
+            const serverName = document.title.split(" | ");
+            presenceData.details = `Managing ${serverName[0]}`;
+            presenceData.state = "Watching Console and hunting bugs!";
+        } else if (document.location.pathname.includes("account")) {
+            presenceData.details = "Account Settings";
+            presenceData.state = "Changing Information";
+        }
+    } else if (document.location.hostname === "status.shardhosting.co.uk") {
+        presenceData.smallImageKey = "hetrix";
+        presenceData.smallImageText = "HetrixTools";
+        presenceData.details = "Server Status";
+        presenceData.state = "Guess what! Someone broke it! Blame Ashley!";
+    } else if (document.location.hostname === "web-ore-01.shardhosting.co.uk") {
+        presenceData.smallImageKey = "cpanel";
+        presenceData.smallImageText = "cPanel";
+        presenceData.details = "Web Hosting";
+        presenceData.state = "Managing cPanel";
+    }
+
+    if (!presenceData.details) {
+        presenceData.details = "Server Hosting";
+        presenceData.state = "Starting at $0.75";

```

Comment:
Remove these

Reply:
May I ask why as this presence is based upon [Sparked Host's one](https://github.com/PreMiD/Presences/blob/main/websites/S/Sparked%20Host/presence.ts) which has these lines as well?

Reply:
Will be removed